enterprise/cmd/repo-updater/internal/authz/perms_syncer.go

```diff
@@ -330,6 +336,7 @@ func (s *PermsSyncer) fetchUserPermsViaExternalAccounts(ctx context.Context, use
 
 	byServiceID := s.providersByServiceID()
 	accounts := s.db.UserExternalAccounts()
+	logger := s.logger.Scoped("fetchUserPermsViaExternalServices", "").With(log.Int32("userID", user.ID))
```

Comment:
```suggestion
	logger := s.logger.Scoped("fetchUserPermsViaExternalAccounts", "sync permisisons using external accounts (login connections)").
		With(log.Int32("userID", user.ID))
```

(also we might want to move this further up so that it's easier to sync with function params and name)

Reply:
🚧 fixed 🚧